src/runtime_src/tools/scripts/petalinux.sh

```diff
@@ -49,152 +53,39 @@ ORIGINAL_DIR=$PWD
-if [ -f recipes-core/images/petalinux-image.bbappend ]; then
-	PETALINUX_IMAGE_BBAPPEND=recipes-core/images/petalinux-image.bbappend
-elif [ -f recipes-core/images/petalinux-image-full.bbappend ]; then
-	PETALINUX_IMAGE_BBAPPEND=recipes-core/images/petalinux-image-full.bbappend
-else
-	echo "Not petalinux image .bbappend file in project-spec/meta-user/recipes-core/images/"
-	exit 1;
-fi
-
-echo 'IMAGE_INSTALL_append = " xrt-dev"'            >> $PETALINUX_IMAGE_BBAPPEND
-echo 'IMAGE_INSTALL_append = " mnt-sd"'             >> $PETALINUX_IMAGE_BBAPPEND
-echo 'IMAGE_INSTALL_append = " xrt"'                >> $PETALINUX_IMAGE_BBAPPEND
-echo 'IMAGE_INSTALL_append = " zocl"'               >> $PETALINUX_IMAGE_BBAPPEND
-echo 'IMAGE_INSTALL_append = " opencl-headers-dev"' >> $PETALINUX_IMAGE_BBAPPEND
-echo 'IMAGE_INSTALL_append = " opencl-clhpp-dev"'   >> $PETALINUX_IMAGE_BBAPPEND
-
-echo " * Adding XRT Kernel Node to Device Tree"
-echo "cat ${XRT_REPO_DIR}/src/runtime_src/core/edge/fragments/xlnk_dts_fragment_mpsoc.dts >> recipes-bsp/device-tree/files/system-user.dtsi"
-cat ${XRT_REPO_DIR}/src/runtime_src/core/edge/fragments/xlnk_dts_fragment_mpsoc.dts >> recipes-bsp/device-tree/files/system-user.dtsi
-echo "cat ${ORIGINAL_DIR}/dsa_build/zcu102_fragment.dts >> recipes-bsp/device-tree/files/system-user.dtsi"
-cat ${ORIGINAL_DIR}/dsa_build/zcu102_fragment.dts >> recipes-bsp/device-tree/files/system-user.dtsi
-
-echo " * Configuring the kernel"
-#Configure Linux kernel (default kernel config is good for zocl driver)
-petalinux-config -c kernel --oldconfig
-
-echo " * Configuring rootfs"
-# Configure rootfs, enable below components:
-#   menu -> "user packages" -> xrt
-#   menu -> "user packages" -> mnt-sd
-#   menu -> "user packages" -> xrt-dev
-#   menu -> "user packages" -> zocl
-#   menu -> "user packages" -> opencl-headers-dev
-#   menu -> "user packages" -> opencl-clhpp-dev
-# Saves to: ${PETALINUX_NAME}/project-spec/configs/rootfs_config
-cp ../configs/rootfs_config{,.orig}
-echo 'CONFIG_xrt=y' >> ../configs/rootfs_config
-echo 'CONFIG_mnt-sd=y' >> ../configs/rootfs_config
-echo 'CONFIG_xrt-dev=y' >> ../configs/rootfs_config
-echo 'CONFIG_zocl=y' >> ../configs/rootfs_config
-echo 'CONFIG_opencl-headers-dev=y' >> ../configs/rootfs_config
-echo 'CONFIG_opencl-clhpp-dev=y' >> ../configs/rootfs_config
-petalinux-config -c rootfs --oldconfig
-
-# Build package
-echo " * Performing PetaLinux Build (from: ${PWD})"
-echo "petalinux-build"
-petalinux-build
+# dsa_build.sh will create ${PLATFORM_NAME}/ directory for petalinux project
+${XRT_REPO_DIR}/src/runtime_src/tools/scripts/peta_build.sh ${ORIGINAL_DIR}/dsa_build/${PLATFORM_NAME}.dsa
 
-cd $ORIGINAL_DIR
 echo " * Copying PetaLinux boot files (from: $PWD)"
-cp ./${PETALINUX_NAME}/images/linux/image.ub ${ORIGINAL_DIR}/dsa_build/src/a53/xrt/image/image.ub
+cd $ORIGINAL_DIR
+cp ./${PLATFORM_NAME}/images/linux/image.ub 	${ORIGINAL_DIR}/dsa_build/src/a53/xrt/image/image.ub
```

Comment:
is it ok to have src/a53/xrt path fixed? but this will make it specific for zynqmp